build.gradle.kts

```diff
@@ -102,8 +102,8 @@ fun Project.configurePublishing() {
   if (javadocTask == null && android != null) {
     // create the Android javadoc if needed
     javadocTask = tasks.create("javadoc", Javadoc::class.java) {
-      source = android.sourceSets["main"].java.sourceFiles
-      classpath += project.files(android.getBootClasspath().joinToString(File.pathSeparator))
+      source = android.sourceSets["main"].java.getSourceFiles()
```

Comment:
Any reason not to use the property syntax?

Reply:
I was also confused about this, but it seems to be a result of `AndroidSourceDirectorySet` being converted to Kotlin in `4.1.1` of the Android Gradle Plugin. [Source](https://android.googlesource.com/platform/tools/base/+/studio-master-dev/build-system/gradle-core/src/main/java/com/android/build/gradle/api/AndroidSourceDirectorySet.kt).